Inputs.md

```diff
@@ -53,17 +53,17 @@ let InputWriter: IInputWritr = new InputWritr({
 ## Running Events
 
 To run events, use `callEvent` and `makePipe`.
-`makePipe`  takes in a trigger, code label to get the alias from the event, and a boolean to determine whether the default event should be prevented and
-returns a function to run the triggered event.
+`makePipe`  takes in a trigger and code label to get the alias from the event and returns a function to run the triggered event.
 
 ```typescript
-InputWritr.makePipe("onkeydown", "left", true);
+let leftKeyPipe: () => void = InputWritr.makePipe("onKeyDown", "left", true);
```

Comment:
Since you're no longer documenting the event prevent boolean, no need to pass it in here.